.github/workflows/build_bazel.yml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2023 The StableHLO Authors.
+
+name: Bazel Build
+
+on:
+  pull_request:
+    branches: [ main ]
```

Comment:
I see that the CMake .yml file also has `paths-ignore: ['**.md', 'docs/**']`. Let's add here as well?